lab-materials/lab-env-data/hub-cluster/hub.yml

```diff
@@ -23,23 +23,25 @@ manifests_dir: hubmanifests
 disconnected_operators:
```

Comment:
Here need to change catalog link to

registry.redhat.io/redhat/redhat-operator-index:v4.13-1688429207

built on: GMT: Tuesday, July 4, 2023 12:06:47 AM

Reply:
Great, thanks!